src/main/java/org/kohsuke/github/GHAppInstallation.java

```diff
@@ -45,6 +46,10 @@ public class GHAppInstallation extends GHObject {
     @JsonProperty("repository_selection")
     private GHRepositorySelection repositorySelection;
     private String htmlUrl;
+    @JsonProperty("suspended_at")
+    private Date suspendedAt;
+    @JsonProperty("suspended_by")
+    private GHUser suspendedBy;
```

Comment:
This is conversion in done automatically. 

```suggestion
```
This is conversion in done automatically. 

```suggestion
```
This conversion is done automatically. 
```suggestion
    private Date suspendedAt;
    private GHUser suspendedBy;
```